study-builder/src/main/java/org/broadinstitute/ddp/studybuilder/task/RarexVersion2.java

```diff
@@ -0,0 +1,723 @@
+                "QOL_SELF_HOPELESS", "QOL_SELF_FEEL_FATIGUED", "QOL_SELF_TROUBLE_STARTING", "QOL_SELF_RUN_DOWN",
+                "QOL_SELF_FATIGUED_AVERAGE", "QOL_SELF_SLEEP_QUALITY", "QOL_SELF_SLEEP_REFRESHING", "QOL_SELF_SLEEP_PROBLEM",
+                "QOL_SELF_SLEEP_DIFFICULTY", "QOL_SELF_TROUBLE_LEISURE_ACTV", "QOL_SELF_TROUBLE_FAMILY_ACTV",
+                "QOL_SELF_TROUBLE_DOING_WORK", "QOL_SELF_TROUBLE_FRIENDS_ACTV", "QOL_SELF_PAIN_DAILY_ACTV",
+                "QOL_SELF_PAIN_WORK", "QOL_SELF_PAIN_SOCIAL", "QOL_SELF_PAIN_CHORES", "QOL_SELF_STAIRS", "QOL_SELF_CHORES",
+
+                "QOL_CHILD_SPORT_EXERCISE", "QOL_CHILD_GET_UP_FROM_FLOOR", "QOL_CHILD_WALK_UP_STAIRS",
+                "QOL_CHILD_ABLE_TO_DO_ACTIVITIES", "QOL_CHILD_AWFUL_MIGHT_HAPPEN", "QOL_CHILD_FELT_NERVOUS",
+                "QOL_CHILD_FELT_WORRIED", "QOL_CHILD_FELT_WORRIED_AT_HOME", "QOL_CHILD_FELT_EVRTHNG_WENT_WRONG",
+                "QOL_CHILD_FELT_LONELY", "QOL_CHILD_FELT_SAD", "QOL_CHILD_HARD_TO_HAVE_FUN", "QOL_CHILD_TIRED_TO_KEEP_UP",
+                "QOL_CHILD_GET_TIRED_EASILY", "QOL_CHILD_TOO_TIRED_FOR_SPORT", "QOL_CHILD_TOO_TIRED_TO_ENJOY",
+                "QOL_CHILD_FELT_ACCEPTED", "QOL_CHILD_COUNT_ON_FRIENDS", "QOL_CHILD_FRIENDS_HELP",
+                "QOL_CHILD_WANT_TO_BE_FRIENDS", "QOL_CHILD_TROUBLE_SLEEPING_PAIN", "QOL_CHILD_PAY_ATTENTION_PAIN",
+                "QOL_CHILD_HARD_TO_RUN_PAIN", "QOL_CHILD_HARD_TO_WALK_ONE_BLOCK_PAIN",
+
+                "QOL_PATIENT_SPORT_EXERCISE", "QOL_PATIENT_GET_UP_FROM_FLOOR", "QOL_PATIENT_WALK_UP_STAIRS",
+                "QOL_PATIENT_ABLE_TO_DO_ACTIVITIES", "QOL_PATIENT_AWFUL_MIGHT_HAPPEN", "QOL_PATIENT_FELT_NERVOUS",
+                "QOL_PATIENT_FELT_WORRIED", "QOL_PATIENT_FELT_WORRIED_AT_HOME", "QOL_PATIENT_FELT_EVRTHNG_WENT_WRONG",
+                "QOL_PATIENT_FELT_LONELY", "QOL_PATIENT_FELT_SAD", "QOL_PATIENT_HARD_TO_HAVE_FUN",
+                "QOL_PATIENT_TIRED_TO_KEEP_UP", "QOL_PATIENT_GET_TIRED_EASILY", "QOL_PATIENT_TOO_TIRED_FOR_SPORT",
+                "QOL_PATIENT_TOO_TIRED_TO_ENJOY", "QOL_PATIENT_FELT_ACCEPTED", "QOL_PATIENT_COUNT_ON_FRIENDS",
+                "QOL_PATIENT_FRIENDS_HELP", "QOL_PATIENT_WANT_TO_BE_FRIENDS", "QOL_PATIENT_TROUBLE_SLEEPING_PAIN",
+                "QOL_PATIENT_PAY_ATTENTION_PAIN", "QOL_PATIENT_HARD_TO_RUN_PAIN", "QOL_PATIENT_HARD_TO_WALK_ONE_BLOCK_PAIN"
+        ));
+        addPicklistOption(handle, studyId, stableIds, option, 60);
+        stableIds.addAll(Set.of("QOL_SELF_PAIN_INTENSITY", "QOL_CHILD_PAIN_INTENSITY", "QOL_PATIENT_PAIN_INTENSITY"));
+        addValidation(handle, studyId, stableIds, ruleConfig);
+        addAsteriskToQuestionContentBlocks(qolSelfId, "QUALITY_OF_LIFE");
+        addAsteriskToQuestionContentBlocks(qolChildId, "CHILD_QUALITY_OF_LIFE");
+        addAsteriskToQuestionContentBlocks(qolPatientId, "PATIENT_QUALITY_OF_LIFE");
+        addBlockToActivity(handle, "QUALITY_OF_LIFE", qolSelfId, 7,
```

Comment:
I don't see any new version tags or any new revision ids being used? Shouldn't we be creating new revisions of the activity definitions, or is RareX asking to get this changes in without revisioning?